### PR TITLE
Move all enums.ts files to types.ts

### PR DESCRIPTION
**Pull request by swese44**
just for better consistency with our main repo

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 *.ejs
 package.json
 package-lock.json
+config/styleguide/config.js
```
Comment (unindented) [inline .prettierignore]: Why do we want to avoid this file?

Comment (swese44) [reply .prettierignore]: on commit, prettier kept removing parens around an arrow function's single arg which then fails the build

Comment (swese44) [reply .prettierignore]: updating eslint rule instead

Comment (unindented) [reply .prettierignore]: Ah, forgot we're running eslint on this codebase.

**Pull request merged**